### PR TITLE
Edit of "Turning on analytics"

### PR DESCRIPTION
**Pull request by StephenBarlow**

### CODE REVIEW

File: graph-manager-docs/source/setup-analytics.md [hunk 1]
```diff
@@ -33,47 +32,38 @@ server.listen().then(({ url }) => {
 });
 ```
 
-For advanced configuration options to set up logging, error filtering, and client-aware metrics reporting take a look at our [server documentation](https://www.apollographql.com/docs/apollo-server/features/metrics.html).
-
-## Other servers
+Alternatively, you can assign your API key to the `ENGINE_API_KEY` environment variable in the environment where Apollo Server will run.
```
Comment (zionts) [inline graph-manager-docs/source/setup-analytics.md]: this language seems backwards to me. I think you would assign the `ENGINE_API_KEY` environment variable to your api key (e.g. `service:foo:193401u5j93`), not the other way around.

File: graph-manager-docs/source/setup-analytics.md [hunk 2]
```diff
@@ -33,47 +32,38 @@ server.listen().then(({ url }) => {
...
 
-There are 2 ways to send metrics data from your server to Engine:
+This is the only change required to begin sending traces to Graph Manager. For advanced configuration options, see [Metrics and logging](https://www.apollographql.com/docs/apollo-server/features/metrics/).
```
Comment (zionts) [inline graph-manager-docs/source/setup-analytics.md]: capital L in logging? 🤷‍♂

File: graph-manager-docs/source/setup-analytics.md [hunk 3]
```diff
@@ -33,47 +32,38 @@ server.listen().then(({ url }) => {
...
 
-The reporting endpoint accepts a batch of "traces" encoded as protobuf. Each individual trace represents execution of a single operation, specifically timing and error information of that execution, broken down by field. Each trace also contains context that is operation-specific (e.g. which client an operation was sent from, if the response was fetched from the cache). In addition to the batch of trace details, the metrics report also includes the context within which all operations were executed (e.g. staging vs prod) in a report header.
+As a starting point, we recommend implementing an extension to the GraphQL execution that creates a report with a single trace, as defined in the `Trace` message of [the protobuf schema](https://github.com/apollographql/apollo-server/blob/master/packages/apollo-engine-reporting-protobuf/src/reports.proto#L7). Then, you can batch multiple traces into a single report. We recommend sending batches every 5 to 10 seconds, and limiting each batch to a reasonable size (~4MB).
```
Comment (zionts) [inline graph-manager-docs/source/setup-analytics.md]: Maybe longer, like 20 seconds tbh

File: graph-manager-docs/source/setup-analytics.md [hunk 4]
```diff
@@ -107,29 +91,53 @@ query AuthorForPost {
...
+After your GraphQL server prepares a batch of traces, it should send them to the Graph Manager reporting endpoint, at the following URL:
+
+```
+https://engine-report.apollodata.com/api/ingress/traces
+```
+
+Each batch should be sent as an HTTP POST request. The body of the request can be one of the following:
+
+* A binary serialization of a `FullTracesReport` message
+* A _Gzipped_ binary serialization of a `FullTracesReport` message
```
Comment (zionts) [inline graph-manager-docs/source/setup-analytics.md]: lowercase g

File: graph-manager-docs/source/setup-analytics.md [hunk 5]
```diff
@@ -107,29 +91,53 @@ query AuthorForPost {
...
+
+To authenticate with Graph Manager, each request must include either:
 
-The TypeScript reference implementation uses a default signature method and allows for that signature method to also be overridden by the user. The [implementation of the default](https://github.com/apollographql/apollo-server/blob/master/packages/apollo-engine-reporting/src/signature.ts) drops unused fragments and/or operations, hides String literals, ignores aliases, sorts the tree deterministically, and ignores whitespace differences. We recommend using the same default signature method for consistency across different server runtimes.
+* An `X-Api-Key` header with a valid API key for your graph
+* An `authtoken` cookie with a valid API key for your graph
```
Comment (zionts) [inline graph-manager-docs/source/setup-analytics.md]: we should probably say only graph-level tokens `service:foo:25902u3r90j` are supported

File: graph-manager-docs/source/setup-analytics.md [hunk 6]
```diff
@@ -107,29 +91,53 @@ query AuthorForPost {
...
+### Implementing additional reporting features
 
-### 4. [Optional] Advanced Reporting Features
+The reference TypeScript implementation includes several features that you might want to include in your implementation. All of these features are implemented in the agent itself, and are documented in the interface description for the `EngineReportingOptions` of [the agent](https://github.com/apollographql/apollo-server/blob/master/packages/apollo-engine-reporting/src/agent.ts#L48).
 
-The reference TypeScript implementation also includes several more advanced features which may be worth porting to new implementations. All of these features are implemented in the agent itself and are documented in the interface description for the EngineReportingOptions of [the agent](https://github.com/apollographql/apollo-server/blob/master/packages/apollo-engine-reporting/src/agent.ts#L51).
+For example, the option to send reports immediately is particularly useful for GraphQL servers that run in a serverless environment, like AWS Lambda or Google Cloud Functions.
```
Comment (zionts) [inline graph-manager-docs/source/setup-analytics.md]: can we remove this line? We don't want to recommend that feature actually 😂 or replace it with
> For example, the option to modify variable values is useful in allowing reporting clients to obfuscate potentially private information while maintaining variable information that's useful in debugging.

File: graph-manager-docs/source/setup-analytics.md [hunk 7]
```diff
@@ -107,29 +91,53 @@ query AuthorForPost {
...
 
-For example, the option to send reports immediately may be particularly useful to GraphQL servers running in a serverless environment, like AWS Lambda or Google Cloud Functions.
+Another important feature is the ability to restrict which information is sent to Graph Manager, particularly to avoid reporting personal data. Because personal data most commonly appears in variables and headers, the TypeScript agent offers options for `privateVariables` and `privateHeaders`.
```
Comment (zionts) [inline graph-manager-docs/source/setup-analytics.md]: those options are deprecated and replaced with `sendVariablesValues` and `sendHeaders` now (and oops I rewrote a line like this above)

File: graph-manager-docs/source/setup-analytics.md (hunk 2 above)
Comment (StephenBarlow) [reply graph-manager-docs/source/setup-analytics.md]: Sure looks like it should be, but we don't title case articles currently

**Comment by akmadmin** [comment]
Pls block

**Pull request merged**